src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -209,6 +235,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder('allow_future_params', 'Whether `param` annotation for future method signature are allowed.'))
+                ->setAllowedTypes(['bool'])
+                ->setDefault(true) // @TODO set to `true` on 4.0
```

Comment:
```suggestion
                ->setDefault(false) // @TODO set to `true` on 4.0
```

set up default behaviour to non-bc change

Reply:
there should be no real BC break

Reply:
I fail to see that.

Reply:
I mean which code has commented params :)

Reply:
Symfony.

Reply:
Yes, and this is wanted there.

Reply:
so, change of fixer behaviour is considered a BC.

Reply:
@keradus if Symfony has "future params" and it is desired to have them documented, shouldn't this option be enabled in their ruleset?

Reply:
Nevermind, just checked it's enabled 😅.